CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+#### 3.0.1+1 - 2021-10-14
```

Comment:
FYI `3.0.2` would have been more idiomatic.

https://github.com/dart-lang/sdk/wiki/External-Package-Maintenance#bumping-versions

Reply:
Oh weird. TIL! 

Out of curiosity, when did this rule change? We'd been doing this for years, as had other packages like vm_service, dart_style, async, etc.

Reply:
There was a bunch of confusion years ago about the intended patterns. Some packages only used `+1` version changes for `0.x` releases, while other packages kept using them after the `1.x` release for doc changes.

Reply:
SGTM - glad we've got a clearly documented policy now!